YifanLiao.md

```diff
@@ -1,8 +1,7 @@
-# Week of Apr 4, 2016
+# Week of Apr 18, 2016
 
 ## Firefox
 
-* PTO from Apr 6 to Apr 8
-* Read download UI source code
-* Bug 553869 - XPIProvider's onUpdateFinished uses error codes not defined in AddonManager
- * good first bug
+* Study downloads panel
+* Study places organizer
+* Notes: https://onedrive.live.com/edit.aspx?resid=B9166883E915A272!453&ithint=onenote%2c&app=OneNote&authkey=!AHiYMf0WEL2sOmg
```

Comment:
I can't see the notes; it requires a MS account to login. It will be better if you can find some bugs to help you study.